src/commands/commands.h

```diff
@@ -4,12 +4,23 @@
 * This file is available under the Redis Labs Source Available License Agreement
 */
 
+#pragma once
+
 //------------------------------------------------------------------------------
 // Module Commands
 //------------------------------------------------------------------------------
-
 #include "cmd_query.h"
 #include "cmd_delete.h"
 #include "cmd_explain.h"
 #include "cmd_profile.h"
+#include "cmd_dispatcher.h"
 #include "cmd_bulk_insert.h"
+
+typedef enum {
+	CMD_QUERY,
+	CMD_DELETE,
+	CMD_EXPLAIN,
+    CMD_PROFILE,
+    CMD_BULK_INSERT,
+    CMD_BULK_UNKNOWN
```

Comment:
Inconsistent whitespace
I think it only really makes sense to have UNKNOWN as the first value in an enum, since that's the value it would have if zero-initialized and never updated.